temoa/cli.py

```
@@ -0,0 +1,286 @@
+# =============================================================================
+logger = logging.getLogger(__name__)
+
+
+def _create_output_folder() -> Path:
+    """Create a default time-stamped folder for outputs."""
+    output_path = Path('output_files', datetime.now().strftime('%Y-%m-%d_%H%M%S'))
+    output_path.mkdir(parents=True, exist_ok=True)
+    return output_path
+
+
+def _setup_logging(output_path: Path, debug: bool = False, silent: bool = False) -> None:
```

Ruff (0.14.3)

Boolean-typed positional argument in function definition

(FBT001)